pyproject.toml

```diff
@@ -55,3 +54,24 @@ build-backend = "flit_core.buildapi"
 
 [tool.flit.sdist]
 include = [".sqlfluff"]
+
+[tool.ruff]
+target-version = "py310"
+
+lint.select = [
+    "A",  # prevent using keywords that clobber python builtins
+    "B",  # bugbear: security warnings
+    "E",  # pycodestyle
+    "F",  # pyflakes
+    "I",  # isort
+    "ISC",  # implicit string concatenation
+    "PLE",  # pylint errors
+    "RUF",  # the ruff developer's own rules
+    "UP",  # alert you when better syntax is available in your python version
+]
+lint.ignore = [
+    "ISC001"  # Recommended ingore from `ruff format`
```

Comment:
Let's keep the gore out plz

But also - why is it recommended to ignore this? Inline concatenation is pretty rare / odd to see. I'm not fighting for this rule, just curious.

Reply:
Ah so it's about conflicts between the formatter and the linter. Sure - maybe expand the comment to link that ticket or say something along those lines. Because this would hopefully be a temporary ignore, until `ruff` fixes itself.

Reply:
Expanded this comment.